### PR TITLE
fix: fix hooks and add tests

### PR DESCRIPTION
**Pull request by puneet2019**
- after epoch hooks was using liquidstakeibc feature instead of liquidstake feature. 
- should not cause problems in prod, because both configs are expected to ideally be same. 
- better to fix as functionality-wise it is buggy.

### CODE REVIEW

**Review by xlab — DISMISSED** [review]

File: x/ratesync/keeper/hooks.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 	for _, hc := range hcs {
 		if hc.Features.LiquidStake.Enabled && epochIdentifier == types.LiquidStakeEpoch {
 			// Add liquidstakekeeper and do stuff
-			err := k.ExecuteLiquidStakeRateTx(ctx, hc.Features.LiquidStakeIBC, liquidBondDenom, bondDenom, nas.MintRate, hc.ID, hc.ConnectionID, hc.ICAAccount)
+			err := k.ExecuteLiquidStakeRateTx(ctx, hc.Features.LiquidStake, liquidBondDenom, bondDenom, nas.MintRate, hc.ID, hc.ConnectionID, hc.ICAAccount)
```
Comment (xlab) [inline x/ratesync/keeper/hooks.go:73]: > take LiquidStakeIBC feature instead of LiquidStake

but it's taking LiquidStake instead of LiquidStakeIBC?

Comment (puneet2019) [reply x/ratesync/keeper/hooks.go:73]: yea, it is inside `LiquidStake.Enabled` , so the features have to be `hc.Features.LiquidStake`

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+### Bug Fixes
+
+- [719](https://github.com/persistenceOne/pstake-native/pull/719) Fix afterEpoch hooks to take LiquidStakeIBC feature
+  instead of LiquidStake
```
Comment (kruspy) [inline CHANGELOG.md:42]: different lines ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -46,7 +51,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
-- [707](https://github.com/persistenceOne/pstake-native/pull/707) Fix liquidstakeibc redeem edge case for protecting cValue  
+- [707](https://github.com/persistenceOne/pstake-native/pull/707) Fix liquidstakeibc redeem edge case for protecting
+  cValue
```
Comment (kruspy) [inline CHANGELOG.md:55]: same

**Comment by kruspy** [comment]
Just 2 nitpicks, rest LGTM

File: x/ratesync/keeper/hooks.go (hunk 1 above)
Comment (puneet2019) [reply x/ratesync/keeper/hooks.go:73]: oh my bad, changelog is bad

File: CHANGELOG.md (hunk 2 above)
Comment (puneet2019) [reply CHANGELOG.md]: auto format

**Review by kruspy — APPROVED** [review]

**Pull request merged**